modules/mysql/src/main/scala/doobie/mysql/JavaTimeInstances.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2013-2020 Rob Norris and Contributors
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package doobie.mysql
+
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+
+import doobie.Meta
+import doobie.enumerated.{JdbcType => JT}
+import doobie.util.meta.MetaConstructors
+
+trait JavaTimeInstances extends MetaConstructors {
+
+  implicit val JavaTimeOffsetDateTimeMeta: Meta[java.time.OffsetDateTime] =
+    Basic.oneObject(
+      JT.Timestamp,
+      List(JT.VarChar, JT.Date, JT.Time),
+      classOf[java.time.OffsetDateTime]
+    )
+
+  implicit val JavaTimeInstantMeta: Meta[java.time.Instant] =
+    JavaTimeOffsetDateTimeMeta.timap(_.toInstant)(OffsetDateTime.ofInstant(_, ZoneOffset.UTC))
+
+  implicit val JavaTimeLocalDateTimeMeta: Meta[java.time.LocalDateTime] =
+    Basic.oneObject(
+      JT.Timestamp,
```

Comment:
MySQL/MariaDB has only `TIMESTAMP` which behaves as if `WITH TIME ZONE`, right? It is the equivalent of Java's `Instant`.

So that would mean that `LocalDateTime` is not suitable for `TIMESTAMP`, right?

Reply:
Yeah those are pretty wrong, should be fixed now.